### DISCUSSION TITLE
Need help with x-bind:name for input field in array

### DISCUSSION THREAD

**Discussion by vik0803** [alpinejs/alpine#2173] [1. Help]
Hello Everyone,

I am trying to use AlpineJS in one of my new projects whereas we have a loop that is creating multiple text inputs and I am using x-bind:name to provide the name to input but I need it in Array which I am not able to figure out. 

I tried various methods but it isnt helping me out so thought to reach out to the community for help. Your help is highly appreciated. 

``<input type="number" x-model="field.impValue[colField.title].value" x-bind:name="colField.title + []" x-text="colField.title" min="0" max="5"  placeholder="Input number between 0-5">``

**Comment by SimoTod** [comment]
If I understood correctly, `colField.title` is a variable that you want to concatenate to the string `[]` (so the output is, for instance, `foo[]`).
In this case, you missed the quotes around the square brackets.

`x-bind:name="colField.title + '[]'"`

**Reply by bhavik-showmgr** [reply]
Thanks @SimoTod It worked.
